scss/edx/overrides/_grid.scss

```diff
@@ -1,6 +1,17 @@
-.container,
+.container {
+  padding-left: $grid-gutter-width;
+  padding-right: $grid-gutter-width;
+}
+
 .container-fluid {
-  max-width: 1140px;
+  max-width: 1140px + 2 * $grid-gutter-width;
   padding-left: $grid-gutter-width;
   padding-right: $grid-gutter-width;
 }
+
+$container-max-widths: map-merge(
+  (
+    xl: 1140px + 2 * $grid-gutter-width,
```

Comment:
Does this also require us to change: `$grid-breakpoints`? https://github.com/edx/paragon/pull/534/files#diff-252b35fc7cc2964fadc678c056766291R109?

Reply:
grid-breakpoints seems to be set to the correct width already, so I leaned on the side of writing less code

Reply:
👍

Reply:
👍